TARGET_STM32L0/TARGET_MTB_MURATA_ABZ/PinNames.h

```diff
@@ -130,34 +130,35 @@ typedef enum {
 #define BUTTON1  PA_3
 
 // Aliases
-#define I2C_SCL     = PB_8,
-#define I2C_SDA     = PB_9,
-
-#define SPI_MOSI    = PB_15,
-#define SPI_MISO    = PB_14,
-#define SPI_SCK     = PB_13,
-
-#define SD_CS       = PB_12,
-
-#define LCD_A0      = PB_2,
-#define LCD_RESET   = PB_5,
-#define LCD_NCS     = PB_6,
-
-#define LORA_SPI_MOSI   = PA_7,
-#define LORA_SPI_MISO   = PA_6,
-#define LORA_SPI_SCLK   = PB_3,
-#define LORA_CS         = PA_15,
-#define LORA_RESET      = PC_0,
-#define LORA_DIO0       = PB_4,
-#define LORA_DIO1       = PB_1,
-#define LORA_DIO2       = PB_0,
-#define LORA_DIO3       = PC_13,
-#define LORA_ANT_RX     = PA_1,
-#define LORA_ANT_TX     = PC_2,
-#define LORA_ANT_BOOST  = PC_1,
-
-#define FLASH_NCS   = PA_8,
-#define POT         = PA_2, // ADC2 - input !
+#define I2C_SCL     PB_8
+#define I2C_SDA     PB_9
+
+#define SPI_MOSI    PB_15
+#define SPI_MISO    PB_14
+#define SPI_SCK     PB_13
+
+#define SD_CS       PB_12
+
+#define LCD_A0      PB_2
+#define LCD_RESET   PB_5
+#define LCD_NCS     PB_6
+
+#define LORA_SPI_MOSI   PA_7
+#define LORA_SPI_MISO   PA_6
+#define LORA_SPI_SCLK   PB_3
+#define LORA_CS         PA_15
+#define LORA_RESET      PC_0
+#define LORA_DIO0       PB_4
+#define LORA_DIO1       PB_1
+#define LORA_DIO2       PB_0
+#define LORA_DIO3       PC_13
+#define LORA_ANT_RX     PA_1
+#define LORA_ANT_TX     PC_2
+#define LORA_ANT_BOOST  PC_1
+#define LORA_VDD_TXCO   NC
```

Comment:
PA_12 ?

Reply:
> PA_12 ?

Really depends on hardware designer, as stated on the readme for [b-l072z-lrwan1](https://www.st.com/en/evaluation-tools/b-l072z-lrwan1.html), there is jumper JP9 on board:

- if position 1-2 then correct `LORA_VDD_TXCO` is `PA_12``
- if position 2-3 then correct `LORA_VDD_TXCO` is `NC`

So I would let the default value, and let user to set PA_12 only if moving the jumper JP9 to that position.